guides/common/modules/proc_booting-virtual-machines.adoc

```diff
@@ -18,13 +18,6 @@ endif::[]
 * You have configured your iPXE environment.
 For more information, see xref:Configuring_iPXE_Environment_{context}[].
 
-[NOTE]
-====
-You can use the original templates shipped in {Project} as described below.
-If you require modification to an original template, clone the template, edit the clone, and associate the clone instead of the original template.
-For more information, see xref:running-custom-code-during-host-provisioning-by-using-web-ui[].
```

Comment:
I think this isn't a correct link for the clone guide, which should be either https://docs.theforeman.org/nightly/Provisioning_Hosts/index-katello.html#cloning-provisioning-templates, right?
cc @stejskalleos @ShimShtein

Reply:
Why is this relevant when we're removing it?